### DISCUSSION TITLE
Search Panel Bug

### DISCUSSION THREAD

**Discussion by Ce1ling** [community/community#70699] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Under `MacOS` Chinese input method:

I use `Cmd + K` to open the search panel, then type some text (e.g.: react) and press `Enter` and it goes straight to search. But I just want to submit the input.  

However, the same action, where I use `/` to open the top search box, does not happen.

The problem is as shown in the picture:

![image](https://github.com/community/community/assets/87267984/dbd37db4-045b-456a-9fd1-75a450dedf06)

In this case if I press `Enter` it triggers the search and I just want to submit the content.

**Comment by jarlefosen** [comment]
Similar issue here, I am unable to open the search menu by clicking `/`.
I'm on a mac with a Norwegian keyboard layout. In order to write `/` I have to press `shift + 7`.

I did however learn that clicking `s` on the keyboard does open the search menu.
<img width="378" alt="image" src="https://github.com/community/community/assets/2993240/a454a41a-c540-47e1-9c61-da36cf839e19">


Surprisingly the slash do in fact trigger commands in the markdown editor.
<img width="482" alt="image" src="https://github.com/community/community/assets/2993240/7b409149-0a67-4e8c-b6e8-73125219c4c6">
